tools/install_prereqs.sh

```diff
@@ -52,7 +52,8 @@ else
     python3-dev \
     python3-pip \
     rocblas-dev \
-    rocm-cmake
+    rocm-cmake \
+    libtbb-dev
```

Comment:
Are we now not worried about other stuff breaking with libtbb-dev? I think its moved to onetbb right now

Reply:
My original concern was the SLES docker and future other dockers I saw with CentOS. I think they don't have a libtbb package but something else.

Its fine. Looks like your docker passed, we can cross the CentOS issues later.